qml/PingStatus.qml

```diff
@@ -62,15 +62,17 @@ Item {
             Label {
                 text: baseModel.title
             }
+            //We disable the models with visibility to avoid unnecessary CPU usage in realocation
+            //TODO: We should improve it to remove the amount of CPU and realocation when visible
```

Comment:
what is the reallocation that is happening? it looks like the model is only updated once on L14. could we use assignment instead of binding?

Reply:
property var sensorModel = ping.sensorStatusModel(root)

Reply:
The problem is that the properties changes, each profile message results in 8 signals, in 50 Hz this results in 400 models constructions.  It appears that Qt delete and create a new model each time.

Reply:
> It appears that Qt delete and create a new model each time.

that is funny.